content/docs/introduction/faq.md

```diff
@@ -221,6 +221,11 @@ second, will only run into precision issues after over 285 years.
 
 ### Why don't the Prometheus server components support TLS or authentication? Can I add those?
 
+Note: The Prometheus team has recently changed their stance on this and
+support for TLS and authentication in serving endpoints is now on the
+[project's roadmap](/docs/introduction/roadmap/#tls-and-authentication-in-http-serving-endpoints).
+As long as this is not implemented yet, the advice below still applies.
```

Comment:
Even when implemented, much of the below still applies.

Reply:
"This document will be updated once code changes have been made."?

Reply:
Replaced with @RichiH's suggestion.